src/components/marker-chart/index.js

```diff
@@ -61,6 +59,19 @@ type Props = ConnectedProps<{||}, StateProps, DispatchProps>;
 
 class MarkerChart extends React.PureComponent<Props> {
   _viewport: HTMLDivElement | null = null;
+
+  /**
+   * Determine the maximum zoom of the viewport.
+   */
+  getMaximumZoom(): UnitIntervalOfProfileRange {
+    const {
+      timeRange: { start, end },
+    } = this.props;
+
+    const ONE_NS = 1e-6;
```

Comment:
It might be worth noting in a comment that setting it to 0 can make the viewport get stuck when zooming in all the way, since it's come up a few times now.